lib/Pages/Teleblitz/home_page.dart

```diff
@@ -198,10 +198,7 @@ class HomePageState extends State<HomePage> with TickerProviderStateMixin {
 
   @override
   void dispose() {
-    moreaLoading.dispose();
-    if(_formType == FormType.leiter){
-      teleblitz.dispose();
```

Comment:
das söttsch aber glaub ich dinnelah, suscht wird doch das WerChunnt nie disposed.

Reply:
Wenn d refrence vo ere klass glöst wird, wird alles glöst. Also wenn mer uf e anderi Page gaht, söttis sowiso alles lösche. https://stackoverflow.com/questions/20482596/how-do-we-delete-an-object-in-dart